code/game/objects/items/weapons/cards_ids.dm

```diff
@@ -181,18 +179,6 @@ var/const/NO_EMAG_ACT = -50
 	var/detail_color
 	var/extra_details
 
-/obj/item/card/id/Initialize()
-	.=..()
-	if(job_access_type)
-		var/datum/job/j = SSjobs.get_by_path(job_access_type)
-		if(j)
-			rank = j.title
-			assignment = rank
-			access |= j.get_access()
-			if(!detail_color)
-				detail_color = j.selection_color
-	update_icon()
```

Comment:
Should still call update_icon as it's indepedant of access setting and needed by stuff like the captain's spare card.